builtin/providers/aws/resource_aws_spot_instance_request_test.go

```diff
@@ -249,3 +307,44 @@ resource "aws_spot_instance_request" "foo_VPC" {
 	}
 }
 `
+
+const testAccAWSSpotInstanceRequestConfig_SubnetAndSG = `
+resource "aws_spot_instance_request" "foo" {
+  ami                         = "ami-6f6d635f"
+  spot_price                  = "0.05"
+  instance_type               = "t1.micro"
```

Comment:
Minor nit: this will probably fulfill faster using a current generation instance like t2.micro.